core/client/src/main/java/alluxio/client/block/BlockWorkerClient.java

```diff
@@ -11,19 +11,18 @@
 
 package alluxio.client.block;
 
-import alluxio.Client;
 import alluxio.exception.AlluxioException;
-import alluxio.exception.ConnectionFailedException;
 import alluxio.wire.LockBlockResult;
 import alluxio.wire.WorkerNetAddress;
 
+import java.io.Closeable;
 import java.io.IOException;
 import java.net.InetSocketAddress;
 
 /**
  * Interface for an Alluxio block worker client.
  */
-public interface BlockWorkerClient extends Client {
+public interface BlockWorkerClient extends Closeable {
```

Comment:
the Client interface we have now is a concept of connection (it has 4 functions: connect(), resetConnection(), isConnected() and close()). This is not applicable after this PR. After this PR, the BlockWorkerClient is just an interface that implements the some functions (e.g. accessBlock). It doesn't own connection anymore.